pyke/targetpixelfile.py

```diff
@@ -250,9 +253,9 @@ def centroids(self, aperture_mask='pipeline'):
         col_centr, row_centr : tuple
             Arrays containing centroids for column and row at each cadence
         """
-        if aperture_mask == 'pipeline':
+        if aperture_mask is 'pipeline':
```

Comment:
this is not correct, should be `==`. `is` checks if `aperture_mask` and 'pipeline' are the same `object`, whereas `==` checks for the value.